isthmus/src/main/java/io/substrait/isthmus/SubstraitRelNodeConverter.java

```diff
@@ -1,21 +1,30 @@
 package io.substrait.isthmus;
 
 import static io.substrait.isthmus.SqlToSubstrait.EXTENSION_COLLECTION;
+import static io.substrait.isthmus.expression.FunctionMappings.SQL_OPERATOR_APPROX_COUNT_DISTINCT;
 
 import io.substrait.function.SimpleExtension;
+import io.substrait.isthmus.expression.AggregateFunctionConverter;
 import io.substrait.isthmus.expression.ExpressionRexConverter;
 import io.substrait.isthmus.expression.ScalarFunctionConverter;
-import io.substrait.relation.AbstractRelVisitor;
-import io.substrait.relation.Filter;
-import io.substrait.relation.NamedScan;
-import io.substrait.relation.Project;
-import io.substrait.relation.Rel;
+import io.substrait.proto.AggregateFunction;
+import io.substrait.relation.*;
```

Comment:
Minor nit - remove wildcard import. You may want to increase default in IntelliJ which annoyingly defaults to this after three imports from the same package.

Reply:
done